authentik/root/settings.py

```diff
@@ -193,9 +192,9 @@
 
 CACHES = {
     "default": {
-        "BACKEND": "django_redis.cache.RedisCache",
-        "LOCATION": f"{_redis_url}/{CONFIG.get('redis.db')}",
-        "TIMEOUT": CONFIG.get_int("redis.cache_timeout", 300),
+        "BACKEND": CONFIG.get("cache.backend", "django_redis.cache.RedisCache"),
```

Comment:
I'm not sure we want to expose this as a config option as we only have the one backend?

Reply:
This gives the user the freedom to choose and use whatever backend might come in handy.
Authentik can still only offer support for Redis. For example one might want to use a database as cache instead.

Reply:
I'm tempted to point people to https://goauthentik.io/docs/installation/configuration#custom-python-settings for this instead of having that in the native config options (mainly going by only exposing config options where we support all possible options)